modules/halloy/hm.nix

```diff
@@ -0,0 +1,18 @@
+{ config, lib, ... }:
+
+let
+  themeFile = config.lib.stylix.colors {
+    template = ./template.mustache;
+    extension = ".toml";
+  };
+in
+{
+  options.stylix.targets.halloy.enable =
+    config.lib.stylix.mkEnableTarget "Halloy" true;
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.halloy.enable)
+      {
+        xdg.configFile."halloy/themes/Stylix.toml".source = themeFile;
```

Comment:
Does the following simplification work:

```suggestion
        xdg.configFile."halloy/themes/stylix.toml".source = themeFile;
```

Reply:
The client reads the theme's name from the file name, shown in the table below. Because of that, I preferred to use uppercase to maintain consistency with the built-in theme naming.
|Lowercase (stylix.toml)|Uppercase (Stylix.toml)|
|---|---|
|![2025-02-07_11-43-40](https://github.com/user-attachments/assets/fb821ba3-400e-4fad-9aa6-99210c46ed18)|![2025-02-07_11-40-10](https://github.com/user-attachments/assets/1afcd727-92d8-42b0-aaa2-ecaa2ab2d237)|